src/lib/components/Feedback/index.js

```diff
@@ -6,18 +6,51 @@ import {BaseElement} from '../BaseElement';
 import {trackEvent} from '../../analytics';
 import './_styles.scss';
 
+/** @types {WebFeedback.Question[]} */
+const questions = [
+  {text: 'Was this page helpful?', label: 'helpful', name: 'Helpfulness'},
+  {
+    text: 'Did this page help you complete your goal(s)?',
+    label: 'effective',
+    name: 'Effectiveness',
+  },
+  {
+    text: 'Did this page have the information you needed?',
+    label: 'complete',
+    name: 'Completeness',
+  },
+  {
+    text: "Was this page's information accurate?",
+    label: 'accuracy',
+    name: 'Accuracy',
+  },
+  {text: 'Was this page easy to read?', label: 'readable', name: 'Readability'},
```

Comment:
```suggestion
```

I'm worried about the feedback widget getting ridiculously long. I'm OK with dropping these questions. Getting signals about whether the page was helpful and whether it helped them complete their goals are probably the most important feedback we can get. Perhaps we should do it in a separate PR though, because it's not related to this feature? In the future I may change my mind and want to add questions back.